examples/todomvc/src/models/todos.js

```diff
@@ -0,0 +1,48 @@
+import { types, destroy } from 'mobx-state-tree'
+
+const Todo = types.model(
+  {
+    text: 'Learn Redux',
+    completed: false,
+    id: 0
```

Comment:
`types.identifier(types.number)` will be better here, it will result in better reconciliation when applying snapshots etc